duckduckgo/iOS
# apply container background to widgets

Task/Issue URL: https://app.asana.com/0/414709148257752/1205862739369423/f
Tech Design URL:
CC:

**Description**:

When building with Xcode 15 the widgets require a container background.

**Steps to test this PR**:
1. Build and run the app using Xcode 15
2. Add all the widgets to the Home Screen and confirm they render correctly
3. Add Lock Screen widgets and confirm they render correctly



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
